test/TestAbsPath.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8; -*-
+import os
+
+from ansiblelint.cli import abspath
+
+
+def test_expand_path_vars(monkeypatch, tmpdir):
+    test_path = os.path.join(str(tmpdir), 'path')  # str(tmpdir): Issue with Py3.5
```

Comment:
I believe we figured out earlier that using the `tmp_path` fixture should be preferred over `tmpdir`.